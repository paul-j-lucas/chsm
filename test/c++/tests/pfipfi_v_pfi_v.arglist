-c | void (*(*psignal)(int sig, void (*f)(int)))(int) | 0
