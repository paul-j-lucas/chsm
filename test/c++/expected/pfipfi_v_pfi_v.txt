decl="void (*(*$)(int sig, void (*f)(int)))(int)" name="psignal"
